README.md

```diff
@@ -39,78 +39,95 @@
-    Visit the [full documentation](https://semgrep.dev/docs/getting-started/) to learn more.
+### Option 2: Getting started from the CLI
+
+1.  Install Semgrep CLI
+
+```
+# For macOS
+$ brew install semgrep
+
+# For Ubuntu/WSL/Linux/macOS
+$ python3 -m pip install semgrep
+
+# To try Semgrep without installation run via Docker
+$ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
+```
+
+2.  Run `semgrep login` to create your account and login to Semgrep. 
+
+Logging into Semgrep gets you access to: 
+- Semgrep Supply Chain: A depenency scanner that detects reachable vulnerabilities in third party libraries
+- Semgrep Code's Pro rules: 600+ high confidence rules written by Semgrep's security research team
+- Semgrep Code's cross-file analysis engine: An advanced code analysis engine, designed to detect complex vulnerabilities, and reduce false positives
+
+3.  Go to your app's root directory and run `semgrep ci`. This will scan your project to check for vulnerabilities in your source code and its dependencies.
+
 
 ### Semgrep Ecosystem
 
 The Semgrep ecosystem includes the following products:
 
-- Semgrep OSS Engine - The open-source engine at the heart of everything (this project).
-- [Semgrep Cloud Platform (SCP)](https://semgrep.dev/login) - Deploy, manage, and monitor SAST and SCA at scale using Semgrep, with [free and paid tiers](https://semgrep.dev/pricing). Integrates with continuous integration (CI) providers such as GitHub, GitLab, CircleCI, and more.
 - [Semgrep Code](https://semgrep.dev/products/semgrep-code) - Scan your code with Semgrep's Pro rules and Semgrep Pro Engine to find OWASP Top 10 vulnerabilities and protect against critical security risks specific to your organization. Semgrep Code is available on both [free and paid tiers](https://semgrep.dev/pricing).
 - [Semgrep Supply Chain (SSC)](https://semgrep.dev/products/semgrep-supply-chain) - A high-signal dependency scanner that detects reachable vulnerabilities in open source third-party libraries and functions across the software development life cycle (SDLC). Semgrep Supply Chain is available on both [free and paid tiers](https://semgrep.dev/pricing).
+- [Semgrep Secrets)](https://semgrep.dev/blog/2023/introducing-semgrep-secrets/)[NEW!] - Secrets detection that uses semantic analysis, improved entropy analysis, and validation together to accurately detect sensitive credentials in developer workflows. Sign up for our [private beta](https://get.semgrep.dev/secrets-beta-request.html) to request early access to the product.
+- [Semgrep Cloud Platform (SCP)](https://semgrep.dev/login) - Deploy, manage, and monitor Semgrep at scale, with [free and paid tiers](https://semgrep.dev/pricing). Integrates with continuous integration (CI) providers such as GitHub, GitLab, CircleCI, and more.
+- Semgrep OSS Engine - The open-source engine at the heart of everything (this project).
+
```

Comment:
I think we should either add more color to Pro Rules and Pro Engine or link to the relevant places. 

One option - 
Semgrep Code - Scan your code with Semgrep's proprietary rules (written by our Security Research team) using our interfile and inter-procedural analysis capabilities. 
Semgrep OSS - The open-source engine and community-contributed rules at the heart of everything (this project).